gc/base/standard/Scavenger.cpp

```diff
@@ -4792,6 +4790,11 @@ MM_Scavenger::internalGarbageCollect(MM_EnvironmentBase *envBase, MM_MemorySubSp
 #endif
 	{
 		mainThreadGarbageCollect(env, allocDescription);
+		if (isBackOutFlagRaised()) {
```

Comment:
put a comment in front of if, something like (feel free to reword):
we want to nest percolate global right into this scavenge, so that excessive GC correctly determines the outer GC and which is the relevant one for calculating excessive GC metrics

Reply:
Added comment in the newest commit.